cpp/dolfinx/io/XDMFFile.h

```diff
@@ -164,6 +164,16 @@ class XDMFFile
                       const mesh::Geometry<T>& x, std::string geometry_xpath,
                       std::string xpath = "/Xdmf/Domain");
 
+  /// Read MeshTags by name
+  /// @param[in] mesh The Mesh that the data is defined on
+  /// @param[in] name
```

Comment:
What is `name`? Can you add a doctstring?

Reply:
Done, but for the record this was already undocumented.